packages/web-app/web-app-front/src/pages/VideosPage/VideoCard/VideoCard.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { Link } from "react-router-dom";
+
+type VideoCardType = {
+  link: string;
+  name: string
+}
+
+export const VideoCard = (props: VideoCardType) => {
+  const { link, name } = props;
+  return (
+    <Link to={link} className="card">
+      <img src="https://via.placeholder.com/140x100" className="card-img-top" alt="img" />
```

Comment:
hard src link to placeholder is not accepted. Please add any stub image locally.